sql_generators/search/templates/fenix_metrics.template.sql

```diff
@@ -19,6 +19,33 @@ metrics_{{ namespace }} AS (
     '{{ channel }}' AS channel,
     normalized_os AS os,
     client_info.android_sdk_version AS os_version,
+    CASE
+    WHEN
+      ARRAY_AGG(metrics.string.metrics_adjust_network)[SAFE_OFFSET(0)] NOT IN (
```

Comment:
Current test failure is related to this part, this is an aggregation so you'll have to add a group-by and potentially other aggregations here